src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -103,7 +103,7 @@ class ValidationStep extends React.Component {
      */
     filterInput(amount) {
         let value = amount ? amount.toString().trim() : '';
-        if (value === '' || !Math.abs(Str.fromUSDToNumber(value)) || _.isNaN(Number(value))) {
+        if (value === '' || _.isNaN(Number(value)) || !Math.abs(Str.fromUSDToNumber(value))) {
```

Comment:
Change the order so we don't even try calling `Str.fromUSDToNumber` with something that is not a number